### ISSUE TITLE
Script showing "fatal: destination path X already exists and is not an empty directory." error on Kali linux

### ISSUE THREAD

**Issue by marcos-borges** [MISP/MISP#8166]
After removing the following lines the script does not show this error:
https://github.com/MISP/MISP/blob/4abfac0d3b9cc519aa582f86cdf66f0dc1e1817f/INSTALL/INSTALL.sh#L3269
https://github.com/MISP/MISP/blob/4abfac0d3b9cc519aa582f86cdf66f0dc1e1817f/INSTALL/INSTALL.sh#L3270
https://github.com/MISP/MISP/blob/4abfac0d3b9cc519aa582f86cdf66f0dc1e1817f/INSTALL/INSTALL.sh#L3271
https://github.com/MISP/MISP/blob/4abfac0d3b9cc519aa582f86cdf66f0dc1e1817f/INSTALL/INSTALL.sh#L3272

Basically on line 3259 you already call git clone on the full branch of MISP, and python-cybox, python-stix, mixbox and python-maec are already copy at this point, so no future git clone is needed for these specifics.

**Comment by Felgastank** [comment]
Yes then what do I do next
